src/components/Notifications/ActionComponent.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import { BellIcon, EnvelopeIcon, SlackHashIcon } from '@patternfly/react-icons';
```

Comment:
I though this was done already (grabbing individual code) but looks like it doesn't... I wonder if there are other ways to avoid the full-syntax per import.